tests/test_consumer/test_extractors/test_extractors_batch_airflow.py

```diff
@@ -66,7 +67,7 @@ def airflow_dag_run_event_start() -> OpenLineageRunEvent:
                     openlineageAdapterVersion=Version("1.10.0"),
                 ),
                 airflowDagRun=OpenLineageAirflowDagRunFacet(
-                    dag=OpenLineageAirflowDagInfo(dag_id="mydag"),
+                    dag=OpenLineageAirflowDagInfo(dag_id="mydag", owner="myuser"),
```

Comment:
Please add unit test for dag/task without owner and with owner=="airflow"

Reply:
- add tests without owner for task/dag and make owner optional in dag facet
- every test in test_extractors_run has owner == "airflow"

Reply:
add tests with 'myuser'